### ISSUE TITLE
Maven Central Lags Behind Artifactory Releases

### ISSUE THREAD

**Issue by AugustNagro** [cincheo/jsweet#665]
Hello,

I've just started trying this library out. I see that the [maven central artifacts](https://search.maven.org/artifact/org.jsweet/jsweet-transpiler) are lagging behind the artifactory versions.

Have you stopped using maven centeral, and may I ask why?

**Comment by lgrignon** [comment]
The deploy process is manual for now. We only deploy some major versions.
You can declare JSweet repositories, as shown in our multiple examples.
Is this something blocking for you? What would you suggest?

**Comment by AugustNagro** [comment]
I do prefer central since signing is required, but I understand the process can be time consuming. (don't change just for me).

Off Topic:

I did get a small example working with the 3.0.0 maven plugin, however :watch compilation was too slow (I imagine it is not incremental yet). It also held a core to 100% CPU even when no changes were made.

If you guys improve the compilation speed and someday want a live-reloading dev server, feel free to cherry-pick the 0-dependency DevServer class I've made for the J2CL library: https://github.com/Vertispan/j2clmavenplugin/commit/67eba4c3ee02b7ef281fd22cb2788c14bfb560cf#diff-4e256e67b893f7a4096e74499415b335da1218cebe3e8a70465769088362eaae
